test/test_videoapi.py

```diff
@@ -177,7 +178,10 @@ def test_accurateseek_middle(self):
                 assert (lb <= frame["pts"]) and (ub >= frame["pts"])
 
     def test_fate_suite(self):
-        video_path = fate("sub/MovText_capability_tester.mp4", VIDEO_DIR)
+        try:
+            video_path = fate("sub/MovText_capability_tester.mp4", VIDEO_DIR)
+        except (urllib.error.URLError, ConnectionError) as error:
```

Comment:
Could you please add a TODO or a FIXME so we remember to remove this eventually? Otherwise LGTM, thanks